### PR TITLE
Attempt to work around GitHub Copilot breaking integration tests

### PR DESCRIPTION
**Pull request by jasonmalinowski**
When the GitHub Copilot extension launches, it may create an output window and asynchronously steal focus. This disables the package on launch so it can't interfere.

### CODE REVIEW

**Review by CyrusNajmabadi — APPROVED** [review]

File: src/VisualStudio/IntegrationTest/TestUtilities/InProcess/InProcComponent.cs [hunk 1]
```diff
@@ -124,10 +124,29 @@ protected static void WaitForApplicationIdle(TimeSpan timeout)
...
 
         protected static void WaitForSystemIdle()
+        {
 #pragma warning disable VSTHRD001 // Avoid legacy thread switching APIs
-            => CurrentApplicationDispatcher.Invoke(() => { }, DispatcherPriority.SystemIdle);
+            CurrentApplicationDispatcher.Invoke(() => { }, DispatcherPriority.SystemIdle);
 #pragma warning restore VSTHRD001 // Avoid legacy thread switching APIs
 
+            // Work around GitHub Copilot creating output windows and stealing focus
+            JoinableTaskFactory.Run(async () =>
```
Comment (sharwell) [inline src/VisualStudio/IntegrationTest/TestUtilities/InProcess/InProcComponent.cs:133]: 📝 For the old integration test framework, I would relocate this code here:

https://github.com/dotnet/roslyn/blob/cb3137e1e5a79c34c5ad99d9532f164893b0a3f8/src/VisualStudio/IntegrationTest/TestSetup/IntegrationTestServicePackage.cs#L19-L32

**Review by sharwell — CHANGES_REQUESTED** [review]

File: src/VisualStudio/IntegrationTest/TestUtilities/InProcess/InProcComponent.cs (hunk 1 above)
Comment (jasonmalinowski) [reply src/VisualStudio/IntegrationTest/TestUtilities/InProcess/InProcComponent.cs:133]: Ah excellent; I was trying to find something similar but couldn't.

Comment (jasonmalinowski) [reply src/VisualStudio/IntegrationTest/TestUtilities/InProcess/InProcComponent.cs]: Going a different direction, so resolving this for now.

**Comment by arkalyanms** [comment]
Can revert https://github.com/dotnet/roslyn/pull/70546 if this works.

File: src/VisualStudio/IntegrationTest/TestUtilities/InProcess/InProcComponent.cs (hunk 1 above)
Comment (jasonmalinowski) [reply src/VisualStudio/IntegrationTest/TestUtilities/InProcess/InProcComponent.cs:134]: This in the end wasn't useful -- it appeared that this package load is happening under a synchronous block due to the command fetch, so in the end so "let the product just run until the output window is opened" isn't something that can be sanely done. So leaving it this way for now, not that I wouldn't have preferred this.

**Pull request merged**